go/mysql/server.go

```diff
@@ -55,6 +55,7 @@ var (
 	timings    = stats.NewTimings("MysqlServerTimings", "MySQL server timings", "operation")
 	connCount  = stats.NewGauge("MysqlServerConnCount", "Active MySQL server connections")
 	connAccept = stats.NewCounter("MysqlServerConnAccepted", "Connections accepted by MySQL server")
+	connClose  = stats.NewCounter("MysqlServerConnClosed", "Connections closed by MySQL server")
```

Comment:
Connections that are closed correctly are accounted. I think we should be explicit about the fact that this is a connection that was closed in a somewhat abrupt way. Maybe we should rename this to be ConnAborted or something like that??

Reply:
or `ConnRefused`?

Reply:
`ConnRefused` sounds good to me!

Reply:
Agreed -- PR has been updated to have more precise naming.